sdk/src/network-client.ts

```diff
@@ -715,7 +799,24 @@ class AleoNetworkClient {
     try {
       return await this.fetchData<Array<ConfirmedTransactionJSON>>("/block/" + height.toString() + "/transactions");
     } catch (error) {
-      throw new Error("Error fetching transactions. " + error);
+      throw new Error(`Error fetching transactions: ${error}`);
+    }
+  }
+
+  /**
+   * Returns the transactions present in the block with the specified hash.
```

Comment:
When speaking about hashes, be specific which one.

```suggestion
   * Returns the confirmed transactions present in the block with the specified block hash.
```